projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -103,11 +108,52 @@ export const getSummaryResourceProps = resourceSlug => {
 				ctaLabel: __( 'Read More', 'jetpack' ),
 				ctaLink: getRedirectUrl( 'jetpack-blog-spam-comments' ),
 			};
+		case 'server-credentials':
+			return {
+				displayName: __( 'Server Credentials', 'jetpack' ),
+				ctaLabel: __( 'Add', 'jetpack' ),
+				ctaLink: getJetpackCloudUrl( state, 'settings' ),
+			};
 		default:
 			throw `Unknown resource slug in getSummaryResourceProps() recommendations/feature-utils.js: ${ resourceSlug }`;
 	}
 };
 
+export const getSummaryPrimaryProps = ( state, primaryProps ) => {
+	switch ( primaryProps ) {
+		case 'backup-activated':
+			return {
+				displayName: __( 'Real-time Backups', 'jetpack' ),
+				ctaLabel: __( 'Manage', 'jetpack' ),
+				ctaLink: getJetpackCloudUrl( state, 'backup' ),
+			};
+		case 'scan-activated':
+			return {
+				displayName: __( 'Real-time Malware Scanning', 'jetpack' ),
+				ctaLabel: __( 'Manage', 'jetpack' ),
+				ctaLink: getJetpackCloudUrl( state, 'scan' ),
+			};
+		case 'antispam-activated':
+			return {
+				displayName: __( 'Automated Spam Protection', 'jetpack' ),
+				ctaLabel: __( 'Manage', 'jetpack' ),
+				ctaLink: getSiteAdminUrl( state ) + 'admin.php?page=akismet-key-config',
+			};
+		case 'videopress-activated':
+			return {
+				displayName: __( 'Ad-free, Customizable Video', 'jetpack' ),
```

Comment:
I'll ask the design team about that! I think it both has sense but also I understand that it can be undesired here (because it's basically a shortcut to the other page where you can get to know more).